### PR TITLE
Release/0.2.0

### PR DESCRIPTION
**Pull request by chuwy**
Hey @alexanderdean!

Regarding #14, I added only two functions as described in ticket to add run id and to check it. However, I'm quite keen to add function to create table as well, as otherwise we need to explain in docs that table must have `RunId` attribute that is a `String` and primary key.

Also I have an implementation that lists all run ids by given path (enriched archive or shredded archive), but decided to opt-out it as you mentioned only two above functions.

Upd: technically, we don't even need boto3 dependency as it is embedded in runtime and not imported anywhere.

### CODE REVIEW

**Comment by alexanderdean** [comment]
Hey @chuwy - 

> However, I'm quite keen to add function to create table as well, as otherwise we need to explain in docs that table must have RunId attribute that is a String and primary key.

Please add it!

> Also I have an implementation that lists all run ids by given path (enriched archive or shredded archive)

Please add it!

> Upd: technically, we don't even need boto3 dependency as it is embedded in runtime and not imported anywhere.

Which runtime embeds it?

**Comment by chuwy** [comment]
> Which runtime embeds it?

@alexanderdean that was just implementation detail you probably not interested in. I don't know why I mentioned it.

**Comment by alexanderdean** [comment]
Can you fix the CI/CD breakage please @chuwy

File: CHANGES.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Migrate setup guide from README to dedicated snowplow/snowplow wiki page (#13)
+Rename CHANGELOG to CHANGES.txt (#8)
+Fix homepage URL (#10)
+Fix author email (#15)
+Update README markdown in according with CommonMark (#11)
+
+Version 0.1.0 (2016-05-17)
+--------------------------
+Initial release
+
```
Comment (alexanderdean) [inline CHANGES.txt]: Remove whitespace at bottom of file

File: README.md [hunk 2]
```diff
@@ -10,78 +10,16 @@ The **[Snowplow][snowplow]** Analytics SDK for Python lets you work with **[Snow
 
 Use this SDK with **[Apache Spark][spark]**, **[AWS Lambda][lambda]**, **[Databricks][databricks]** and other Python-compatible data processing frameworks.
 
-## Functionality
+## Documentation
 
-The Snowplow enriched event is a relatively complex TSV string containing self-describing JSONs. Rather than work with this structure directly in Scala, use this Analytics SDK to interact with the enriched event format:
+[Setup][setup-guide] and [User][user-guide] Guides avaiable at [Snowplow wiki][snowplow-wiki].
```
Comment (alexanderdean) [inline README.md]: Better:

* [Setup guide][setup-guide]
* [User guide][user-guide]
```

File: Vagrantfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.hostname = "snowplow-python-analytics-sdk"
+  config.ssh.forward_agent = true
+
+  config.vm.provider :virtualbox do |vb|
+    vb.name = Dir.pwd().split("/")[-1] + "-" + Time.now.to_f.to_i.to_s
+    vb.customize ["modifyvm", :id, "--natdnshostresolver1", "on"]
+    vb.customize [ "guestproperty", "set", :id, "--timesync-threshold", 10000 ]
+    vb.memory = 2048
```
Comment (alexanderdean) [inline Vagrantfile]: That's quite a lot of RAM no?

**Review by alexanderdean — CHANGES_REQUESTED** [review]
Plus don't forget to add the new methods you mentioned!

**Pull request closed without merging**